src/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/Areas/Identity/Controllers/AccountController.cs

```diff
@@ -0,0 +1,465 @@
+﻿using System;
```

Comment:
I'm assuming this 400-line file is just a direct copy of what we already have, so am not going to review its content.

But if I did review its content I would raise a question about some bits such as "Don't reveal that the user does not exist or is not confirmed" - that seems unhelpful and not aiding security (since someone can determine that by trying to register the account anyway).

Reply:
I will add it to the list of cleanups (I kept a list while I was moving the code of the things that I saw)